com.etlegacy.ETL.yaml

```
@@ -230,19 +233,23 @@ modules:
           - rm enemy-territory.zip
           - rm et260b.x86_keygen_V03.run
```

Maybe make this a wild card so you don't need to update in the future?
```suggestion
          - rm et*.x86_keygen_V03.run
```
```suggestion
          - cp -r ./etlegacy-*-i386/legacy/ .
```

---

It will be unlikely that this file ever changes. It's also only required for old mods: Modern servers don't require it I recall.